tools/archetype/binding/src/main/resources/archetype-resources/ESH-INF/binding/binding.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <binding:binding id="${bindingId}"
-				 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
No, no special purpose. Just wanted to retrigger the validations. I will revert those changes.